# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

ActionMsgMapperActionGoal action_goal
ActionMsgMapperActionResult action_result
ActionMsgMapperActionFeedback action_feedback
